packages/shared/package.json

```diff
@@ -40,6 +40,7 @@
         "@types/object.fromentries": "^2.0.4",
         "@types/react": "^18.0.7",
         "@types/react-dom": "^18.0.0",
+        "@types/ua-parser-js": "^0.7.39",
```

Comment:
Previously @types/ua-parser-js was added as a dependency of accounts-v2.
 ua-parser-js is a dependecy of shared package, hence moved the @types/ua-parser-js  package dependecy to shared package.